AsyncDisplayKit/Details/Transactions/_ASAsyncTransaction.h

```diff
@@ -70,18 +72,18 @@ extern NSInteger const ASDefaultTransactionPriority;
 /**
  The dispatch queue that the completion blocks will be called on.
  */
-@property (nonatomic, readonly, retain) dispatch_queue_t callbackQueue;
+@property (nonatomic, readonly, strong) dispatch_queue_t callbackQueue;
 
 /**
  A block that is called when the transaction is completed.
  */
-@property (nonatomic, readonly, copy) asyncdisplaykit_async_transaction_completion_block_t completionBlock;
+@property (nonatomic, readonly, copy, nullable) asyncdisplaykit_async_transaction_completion_block_t completionBlock;
 
 /**
  The state of the transaction.
  @see ASAsyncTransactionState
  */
-@property (nonatomic, readonly, assign) ASAsyncTransactionState state;
+@property (readonly, assign) ASAsyncTransactionState state;
```

Comment:
No, because we implement the accessors to make it atomic. So removing `nonatomic` here is just documentation.